.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,69 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: 'CodeQL'
+
+on:
+  push:
+    branches: [master, development]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [development]
```

Comment:
Will this still cover pull requests branched off `master` or  LTS fix releases branched off an LTS release branch? Or is this something we'll add later on since this is a test PR (based on the title)?

Reply:
Just changed to run it against all pull requests